exchanges/exchange_test.go

```diff
@@ -2906,6 +2906,60 @@ func TestCanUseAuthenticatedWebsocketEndpoints(t *testing.T) {
 	assert.True(t, e.CanUseAuthenticatedWebsocketEndpoints())
 }
 
+func TestGetCachedTicker(t *testing.T) {
+	t.Parallel()
+	b := Base{Name: "test"}
+	pair := currency.NewPair(currency.BTC, currency.USDT)
+	_, err := b.GetCachedTicker(pair, asset.Spot)
+	assert.ErrorIs(t, err, ticker.ErrNoTickerFound)
+
+	err = ticker.ProcessTicker(&ticker.Price{ExchangeName: "test", Pair: pair, AssetType: asset.Spot})
+	assert.NoError(t, err)
+
+	tickerPrice, err := b.GetCachedTicker(pair, asset.Spot)
+	assert.NoError(t, err)
+	assert.Equal(t, tickerPrice.Pair, pair)
```

Comment:
```suggestion
	assert.Equal(t, pair, tickerPrice.Pair)
```

Reply:
Soz, there were a few others of these below to catch as well, e.g. L2936

Reply:
[changed](https://github.com/thrasher-corp/gocryptotrader/pull/1440/commits/5f7705bc25e38dde3ced076f4bf30b83cc6870dd)

I could not see anymore could you please have a gander where more are if you have some time 🙏